src/CapabilityService.WebApi/Infrastructure/Api/CapabilityController.cs

```diff
@@ -101,6 +101,28 @@ public async Task<IActionResult> UpdateCapability(string id, [FromBody] Capabili
             }
         }
         
+        
+        [HttpDelete("{id}")]
+        public async Task<IActionResult> DeleteCapability(string id)
+        {
+            var capabilityId = Guid.Empty;
+            Guid.TryParse(id, out capabilityId);

```

Comment:
You should probably return NotFound (404) or Unprocessable Entity (422) aftet this line is it evals to false == input cant be parsed.

Reply:
I agree, we can also handle the other returns codes more gracefully
`Task added to todo list`